### DISCUSSION TITLE
Cookbook

### DISCUSSION THREAD

**Discussion by apraga** [papis/papis#910] [Ideas]
Hi,

I think it would be a good addition to have a cookbook to help beginners better exploit/understand the power of papis. 4 ideas so far:

- `papis rename --folder-name "{doc[title]}" --all --batch`
- `p update --auto --all --batch`
- import from bibtex into papis database (not sure about the command)
- whoosh expressions for practical examples

Thanks

**Comment by alexfikl** [comment]
Most commands have a little "Examples" section, so that would be a nice place to add stuff like this. 
https://papis.readthedocs.io/en/latest/commands/bibtex.html#examples

Unfortunately, a lot of them are just "stuff we managed to think about when writing the initial docs", so if you have more practical examples, make a PR! They're very welcome :heart: 

> whoosh expressions for practical examples

This also has some examples here https://papis.readthedocs.io/en/latest/database_structure.html#whoosh-database. Any improvements are very welcome! The docs for the databases haven't been updated in quite a while to my knowledge, so they probably need some love :(

**Comment by apraga** [comment]
Fair point. I was trying to point out that some examples are somewhat « buried » in the commands and that you need to know what to look for. Maybe the tutorial would be a better place to direct new users like me. I’ll think on it and submit a draft :)  Message ID: ***@***.***&gt;

**Comment by jghauser** [comment]
Another option would be to add some things to the Wiki. We once started it exactly for this, to gather various useful snippets/commands/configurations. I'm afraid it's never really taken off though... (and it might not be great to have such help docs in two places anyway). But it might be a good thing to add these things there anyway -- at least they'll be somewhere!
